augurpy/estimators.py

```diff
@@ -70,7 +72,9 @@ def create_estimator(
             ),
         )
         c.case("logistic_regression_classifier", lambda: LogisticRegression(penalty=params.get("penalty", "l2")))
-        c.default(_raise_exception)
+        c.default(_raise_exception)(
+            "Missing valid input. Choose rf or lr for categorical labels and rf for continuous labels."
+        )
```

Comment:
Das hier funktioniert so leider nicht, ich bekomme da immer einen `TypeError: 'NoneType' object is not callable`. Das folgende scheint allerdings zu funktionieren: 
`c.default(lambda: _raise_exception("Missing valid input"))`
Oder lieber die vorherige version?

Reply:
`c.default(lambda: _raise_exception("Missing valid input"))`

This looks fine to me.